Update introduction.md (#23328)
dotnet/AspNetCore.Docs: aspnetcore/signalr/introduction.md

* Collaborative apps. Whiteboard apps and team meeting software are examples of collaborative apps.
* Apps that require notifications. Social networks, email, chat, games, travel alerts, and many other apps use notifications.

> SignalR provides an API for creating server-to-client [remote procedure calls (RPC)](https://wikipedia.org/wiki/Remote_procedure_call). The RPCs invoke functions on clients from server-side .NET Core code. There are various [supported platforms](xref:signalr/supported-platforms), each with thier respective client SDK &mdash; as such the RPC function invocation language varies.

Reviewer: Nice improvement. Per [Writing for translation](https://review.docs.microsoft.com/en-us/help/contribute/localization-mt-guidance?branch=master)  `&mdash;` doesn't MT and we like to avoid it.  
I'll toss out a suggestion, feel free to ignore.

```suggestion
SignalR provides an API for creating server-to-client [remote procedure calls (RPC)](https://wikipedia.org/wiki/Remote_procedure_call). The RPCs invoke functions on clients from server-side .NET Core code. There are several [supported platforms](xref:signalr/supported-platforms), JavaScript being the most commonly language.
```

Let's get [TB12's](https://en.wikipedia.org/wiki/Tom_Brady) opinion.

Author: ```suggestion
SignalR provides an API for creating server-to-client [remote procedure calls (RPC)](https://wikipedia.org/wiki/Remote_procedure_call). The RPCs invoke functions on clients from server-side .NET Core code. There are several [supported platforms](xref:signalr/supported-platforms), each with their respective client SDK. Because of this, the programming language being invoked by the RPC call varies.
```

Reviewer: @IEvangelist  closer. ***this*** is ambiguous. I did a slight tweak of your suggestion. I think JavaScript is used by 90%+, so it might be good to mention that.

> SignalR provides an API for creating server-to-client [remote procedure calls (RPC)](https://wikipedia.org/wiki/Remote_procedure_call). The RPCs invoke functions on clients from server-side .NET Core code. There are several [supported platforms](xref:signalr/supported-platforms), each with their respective client SDK. Because of this, the programming language being invoked by the RPC call varies.

Reviewer 2: 👍
